pipelines/wdl/arrays/imputation_beagle/input_qc/test_inputs/Plumbing/NA12878_qc_fails_bad_order_chr.json

```diff
@@ -0,0 +1,8 @@
+{
+    "ArrayImputationQC.multi_sample_vcf": "gs://broad-gotc-test-storage/imputation_beagle/input_qc/scientific/vcfs/test_qc_not_in_order_chr.vcf.gz",
+    "ArrayImputationQC.ref_dict": "gs://gcp-public-data--broad-references/hg38/v0/Homo_sapiens_assembly38.dict",
+    "ArrayImputationQC.reference_panel_path_prefix": "gs://broad-gotc-test-storage/imputation_beagle/scientific/1000G_HGDP_no_singletons_reference_panel/hgdp.tgp.gwaspy.AN_added.bcf.ac2",
+    "ArrayImputationQC.contigs": ["chr1","chr2","chr3","chr4","chr5","chr6","chr7","chr8","chr9","chr10","chr11","chr12","chr13","chr14","chr15","chr16","chr17","chr18","chr19","chr20","chr21","chr22"],
+    "ArrayImputationQC.genetic_maps_path": "gs://broad-gotc-test-storage/imputation_beagle/scientific/plink-genetic-maps/",
+    "ArrayImputationQC.output_basename": "plumbing_test_fail_bad_order_chr"
```

Comment:
by bad order do we mean not sorted?

Reply:
yes... i was just using your file names, so that's what i took this as (and what the qc output also identifies)